### DISCUSSION TITLE
Load testing using artillery

### DISCUSSION THREAD

**Discussion by shahiakriti** [artilleryio/artillery#1453] [Artillery]
Hi all, 

I am trying to run load test for pre existing test suites I built on my project using playwright. I have added the spec file on my yaml file to run the load test but I am getting the following error 

![image](https://user-images.githubusercontent.com/99622173/169727624-5a0e0edc-ab7a-4ea1-9640-5dca8f284b88.png)

I am fairly new to artillery, any help would be appreciated. Thanks

**Comment by hassy** [comment]
hi @shahiakriti 👋 how are you running those tests with Playwright?

**Reply by shahiakriti** [reply]
Hey I am using the standard playwright command which runs my spec.ts file in my tests folder.
`npx playwright test`

**Reply by hassy** [reply]
Artillery does not support running Typescript code directly. You will need to compile those tests into plain Javascript. This page has more information: https://playwright.dev/docs/test-typescript
